Dockerfile

```diff
@@ -53,53 +52,59 @@ ENV \
-    && mv \
+        jq; \
+    wget -q -O - "http://search.maven.org/remotecontent?filepath=io/prometheus/jmx/jmx_prometheus_javaagent/${PROMETHEUS_VERSION}/jmx_prometheus_javaagent-${PROMETHEUS_VERSION}.jar" > /usr/local/share/prometheus-agent.jar; \
+    wget -q -O - "http://search.maven.org/remotecontent?filepath=org/jolokia/jolokia-jvm/${JOLOKIA_VERSION}/jolokia-jvm-${JOLOKIA_VERSION}-agent.jar" > /usr/local/share/jolokia-agent.jar; \
+    mirror_url=$( wget -q -O - 'https://www.apache.org/dyn/closer.cgi?as_json=1' | jq --raw-output '.preferred' ); \
+    wget -q -O - "${mirror_url}cassandra/${CASSANDRA_VERSION}/apache-cassandra-${CASSANDRA_VERSION}-bin.tar.gz" > /usr/local/apache-cassandra-bin.tar.gz; \
+    tar -xzf /usr/local/apache-cassandra-bin.tar.gz -C /usr/local; \
+    rm /usr/local/apache-cassandra-bin.tar.gz; \
+    ln -s $CASSANDRA_HOME /usr/local/apache-cassandra; \
+    wget -q -O - https://github.com/Yelp/dumb-init/releases/download/v${DI_VERSION}/dumb-init_${DI_VERSION}_amd64 > /sbin/dumb-init; \
+    adduser --disabled-password --no-create-home --gecos '' --disabled-login cassandra; \
+    mkdir -p /var/lib/cassandra/ /var/log/cassandra/ /etc/cassandra/triggers; \
+    chmod +x /sbin/dumb-init /ready-probe.sh; \
+    chown cassandra: /ready-probe.sh; \
+    mv \
       /logback.xml \
       /cassandra.yaml \
       /jvm.options \
       /prometheus.yaml \
       /pre_stop.sh \
-      $CASSANDRA_CONF \
-    && mv /usr/local/apache-cassandra/conf/cassandra-env.sh /etc/cassandra/ \
-    && mkdir -p $CASSANDRA_DATA/data $CASSANDRA_LOGS \
-    && ls -la $CASSANDRA_DATA $CASSANDRA_LOGS \
-    && mkdir -p /files \
-    && touch /files/file.log \
-    && chown -R cassandra: /files $CASSANDRA_CONF $CASSANDRA_DATA $CASSANDRA_LOGS \
-    && chmod 700 $CASSANDRA_DATA \
-    && ls -la $CASSANDRA_DATA $CASSANDRA_LOGS \
-    && chown cassandra: /run.sh \
-    && if [ -n "$CQLSH_CONTAINER" ]; then apt-get -y --no-install-recommends install python; else rm -rf  $CASSANDRA_HOME/pylib; fi \
-    && apt-get update && apt-get -qq -y install \
+      $CASSANDRA_CONF; \
+    mv /usr/local/apache-cassandra/conf/cassandra-env.sh /etc/cassandra/; \
+    mkdir -p $CASSANDRA_DATA/data $CASSANDRA_LOGS; \
+    ls -la $CASSANDRA_DATA $CASSANDRA_LOGS; \
+    mkdir -p /files; \
+    touch /files/file.log; \
+    chown -R cassandra: /files $CASSANDRA_CONF $CASSANDRA_DATA $CASSANDRA_LOGS; \
+    chmod 700 $CASSANDRA_DATA; \
+    ls -la $CASSANDRA_DATA $CASSANDRA_LOGS; \
+    chown cassandra: /run.sh; \
+    mkdir -p /tmp chown cassandra /tmp; \
+    if [ -n "$CQLSH_CONTAINER" ]; then apt-get -y --no-install-recommends install python; else rm -rf  $CASSANDRA_HOME/pylib; fi; \
+    apt-get update && apt-get -qq -y install \
        libcap2-bin \
        procps \
-       dnsutils \
-    && apt-get -y purge wget jq localepurge \
-    && apt-get -y autoremove \
-    && apt-get -y clean \
-    && rm -rf \
+       dnsutils; \
```

Comment:
@allamand Why replacing `&&` by `;` ??? We want it to fail if any statement fails.

Reply:
even with ; it will fail